component/main.jsonnet

```diff
@@ -66,10 +68,23 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
   ],
 };
 
+// adding namespace for syn-appcat
+local ns = kube.Namespace(params.namespace);
+
+local secret = kube.Secret(params.services.vshn.emailAlerting.secretName) {
+  metadata+: {
+    namespace: params.services.vshn.emailAlerting.secretNamespace,
+  },
+  stringData: {
+    password: params.services.vshn.emailAlerting.smtpPassword,
+  },
+};
```

Comment:
Nitpick: I'd prefer if we'd only deploy the secret if we actually need it. I.e. if VSHN services are enabled.

Reply:
Fixed